src/DTO/AbstractDTO.php

```diff
@@ -0,0 +1,6 @@
+<?php
```

Comment:
This Class is only usefull for typing our service interfaces.
(this comment is also a test because i can't find review's comment)